tests/bootstrap.php

```diff
@@ -5,3 +5,10 @@
 require __DIR__ . '/../vendor/autoload.php';
 
 AnnotationRegistry::registerLoader('class_exists');
+
+// Polyfill PHPUnit 6.0 both ways
```

Comment:
Is still really needed? Namespaced TestCase is in phpunit 5.7 already

Reply:
See https://travis-ci.org/SimpleBus/SymfonyBridge/jobs/310934357

Reply:
It fails on SF 2.8 with PHP 7.2

Reply:
We could require symony/phpunit-bridge then, just to not reinvent the wheel